renderdoc/data/glsl/pixelhistory_mscopy_gl.comp

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
```

Comment:
I would rather have this stay in `pixelhistory_mscopy.comp` and `pixelhistory_mscopy_depth.comp` use `#ifdef` to have two different paths as necessary for GL. I think it looks like the only difference that would be needed is swapping push constants for a UBO (or you could use plain uniforms if that's more convenient)?